app/scripts/modules/core/src/managed/overview/Resource.tsx

```diff
@@ -25,6 +25,7 @@ const statusUtils: {
   NOT_MANAGED: { color: 'var(--color-status-warning)', icon: 'fas fa-pause', defaultReason: 'Resource is not managed' },
   WAITING: { icon: 'far fa-hourglass', defaultReason: 'Resource is currently locked and can not be updated' },
   PROCESSING: { icon: 'far fa-hourglass', defaultReason: 'Resource is being updated' },
+  DELETING: { icon: 'fas fa-trash-o', defaultReason: 'Resource is being deleted' },
```

Comment:
@ranihorev I wasn't sure what the difference between `fas` and `far` was here, so I just picked one at random...

Reply:
Check on fontawesome.com (make sure to filter for free icons)

Reply:
I did, but in fontawesome they all just had `fa` as the prefix. 🤷‍♂️

Reply:
Changed it to `far fa-trash-can` (assuming the `r` is for `regular`): https://fontawesome.com/v6.0/icons?d=gallery&p=1&q=trash&s=regular&m=free